packages/drivers/routerlicious-driver/src/documentDeltaConnection.ts

```diff
@@ -107,4 +107,13 @@ export class R11sDocumentDeltaConnection extends DocumentDeltaConnection {
 			url: getUrlForTelemetry(this.url, socketIoPath),
 		};
 	}
+
+	/**
+	 * Disconnect from the websocket
+	 */
+	protected disconnectCore(err: IAnyDriverError): void {
+		// tell the server we are disconnecting this client from the document
+		this.socket.emit("disconnect_document", this.clientId, this.documentId, err);
```

Comment:
Noticed that here, `err: IAnyDriverError` but in the service, it is expected as `err: string`. Is Socket.io automatically stringifying the error? That doesn't seem to be the case for other messages, so this might be a typo or oversight